_posts/2019-07-29-plonk-stack.md

```diff
@@ -0,0 +1,211 @@
+
+You will love the Linkerd UI and dashboard which you can open up with a single command `linkerd dashboard`.
+
+* traffic shifting
+
+One of the newest features of Linkerd 2.4 is the ability to shift traffic proportionally between two services by assigning them both a weight. This works out of the box with OpenFaaS without any additional need for changes to the project.
+
+Linkerd has a very active and welcoming community. Their primary sponsor is [Buoyant](https://buoyant.io), which was founded by [William Morgan](https://twitter.com/wm) and [Oliver Gould](https://twitter.com/olix0r) after leaving Twitter.
+
+There are many other features a service-mesh can offer, but these are my top features based upon their usefulness to a FaaS or PaaS.
+
+#### OpenFaaS
+
+When I started OpenFaaS in 2016, the project was going up against some very successful projects from huge corporations. Oftentimes the corporations were producing products which were far too complex to use and operate on a daily-basis. Even the newest projects in the Serverless space have been widely criticised for trying to do too much and being spread thin.
+
+<blockquote class="twitter-tweet" data-conversation="none"><p lang="en" dir="ltr">Three years in, what&#39;s the mission of <a href="https://twitter.com/openfaas?ref_src=twsrc%5Etfw">@OpenFaaS</a>? <a href="https://t.co/WigsKE4UZ9">pic.twitter.com/WigsKE4UZ9</a></p>&mdash; Alex Ellis (@alexellisuk) <a href="https://twitter.com/alexellisuk/status/1158064708202749954?ref_src=twsrc%5Etfw">August 4, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The motto of OpenFaaS is "Serverless Functions Made Simple" and this is reflected in the project values:
+
+* developers-first
+
+* operationally simple
+
+* community-centric
+
+To date there are dozens of end-user companies, some of which have given permission to list their logo on [OpenFaaS.com](https://openfaas.com), over 230 contributors, 18k GitHub stars, and [hundreds of community blogs and events recorded](https://github.com/openfaas/faas/blob/master/community.md).
+
+If you're new to OpenFaaS and are wondering what it's all about, then you can get up to speed with my video from Goto: Serverless Beyond the Hype:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yOpYYYRuDQ0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+In simple terms OpenFaaS offers:
+
+* an easy way to package any code or binary
+* a rich ecosystem of language templates
+* a function store to collaborate and share
+* metrics, auto-scaling, and dozens of detailed tutorials
+* a native experience on Kubernetes, without the overheads
```

Comment:
I think we can remove the `without the overheads`, the native experience on Kubernetes gives just enough info

Reply:
If we think about people who haven't taken the plunge with K8s yet then `without the overheads` might offer some comfort.